sdk/modelsrepository/Azure.Iot.ModelsRepository/tests/TestHelpers.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using System;
+using System.IO;
+using System.Reflection;
+using System.Text.Json;
+
+namespace Azure.Iot.ModelsRepository.Tests
+{
+    public class TestHelpers
+    {
+        private static readonly string s_fallbackTestRemoteRepo = "https://devicemodels.azure.com/";
```

Comment:
And this is why the constant in the client should probably be public.

Reply:
I am not sure I am following. Can you elaborate?

Reply:
This constant exists in the client, but it is not public. Why not make it public so people can see what the default endpoint is?

Reply:
On ResolverClient -> at the bottom, there is a getter for a Property called RepositoryUri. It's public

Reply:
because you can point to your own repo during the client instantiation and this property will return that value to you

Reply:
to your point, I think it's also worth making the DefaulRepository string public. I will add that.

Reply:
Then why is this test code not referencing it?

FWIW, I think it is weird to have a field constant and a property. I didn't notice the property. They can be collapsed to just a property. There's no reason for a field.

Reply:
I see, there are two things. The one you used, and our default.

Our ADT client doesn't reveal your endpoint. I don't think that is necessary.